dist.ini

```diff
@@ -20,11 +20,7 @@ inherit_version = 0    ;optional flag
 [License]
 [ExtraTests]
 [PodSyntaxTests]
-[PodWeaver]
```

Comment:
I could not replicate problems you might have been having with PodWeaver with a fresh perl 5.26.0 install.